server/handlers/task/update.go

```diff
@@ -0,0 +1,38 @@
+package task
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/labstack/echo"
+	"net/http"
+
+	m "github.com/kirikami/go_exercise_api/database/models"
+	u "github.com/kirikami/go_exercise_api/utils"
+)
+
+func (h ApiV1Handler) UpdateTaskHandler(c echo.Context) error {
+	id, err := u.ParseIdInt64FromString(c.Param("id"))
+
+	if err != nil {
+		u.SendError(http.StatusBadRequest, c, err, IdErrorMessage)
```

Comment:
@Kirikami I would do something like. You have to split user/client errors from internal error.
```
type AppError struct {
UserMessage string
}
.... Implement error interface here and `func NewAppError("Invalid ID parameter")`

Then 


var (
ErrInvalidIDParameter = NewAppError("Invalid ID parameter")
)
```

`return SendBadRequest(c, ErrInvalidIDParameter)`
then
`func SendBadRequest(userError AppError, internalError err) error {
      return NewErrorWIthStatus(http.StatusBadRequest, userError, internalError)
}`

THEN declare custom echo error handler `func EchoErrorHandler(err error, c *echo.Context) {`
```
	appError, ok := err.(AppErrorWithStatus)
	if !ok {
		panic("Invalid response error type")
	}
    // then you can log here internal error and return to the client `userError`

```